### PR TITLE
Include Symfony 2.7 stable on Travis and improve configuration

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Also added `symfony/phpunit-bridge` to handle deprecated issues with ease.

Tests will fail because of deprecation (see #636).

Route config issues are solved on #614. I'll take look for form issues on another PR.

### CODE REVIEW

**Comment by stof** [comment]
You can avoid making test fail because of deprecations by setting the appropriate env variable though

File: .travis.yml [hunk 1]
```diff
@@ -12,27 +12,34 @@ php:
   - 5.6
   - nightly
   - hhvm
+  - hhvm-nightly
```
Comment (stof) [inline .travis.yml]: this should be reverted. hhvm-nightly does not exist anymore on Travis

File: .travis.yml [hunk 2]
```diff
@@ -12,27 +12,34 @@ php:
...
-    allow_failures:
+          env: SYMFONY_VERSION=2.7.*
+        - php: 5.6
+          env: SYMFONY_VERSION=2.8.*@dev
         - php: 5.6
-          env: SYMFONY_VERSION='dev-master'
+          env: SYMFONY_VERSION="3.0.x-dev as 2.8"
+    allow_failures:
         - php: nightly
+        - php: hhvm
```
Comment (stof) [inline .travis.yml]: why allowing failures on hhvm ? it was not allowed previously

**Comment by Soullivaneuh** [comment]
@stof And what is this magic env variable? :-)

**Comment by stof** [comment]
It is ``SYMFONY_DEPRECATIONS_HELPER=weak``, as explained in the readme of the bridge: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bridge/PhpUnit/README.md

**Comment by Soullivaneuh** [comment]
@stof Thanks for your advice.

@willdurand Tests should be green now. Ready to merge. :+1:

**Comment by willdurand** [comment]
thanks!

**Pull request merged**